install/traf_tools_setup.sh

```diff
@@ -260,3 +260,14 @@ make install
 if [ $VERBOSE -eq 1 ]; then
   echo "STEP completed:  installation of THRIFT"
 fi
+
+# install maven, if not available already
+MAVEN_VERSION=`mvn --version 2>/dev/null | grep 'Apache Maven' | cut -f 3 -d ' '`
+if [[ "$MAVEN_VERSION" != "3.0.5" ]]; then
```

Comment:
The current default is maven 3.3.3.  I have been using this version without issue.  Do we really need to check for an older version?